.github/workflows/remove-issue-labels.yaml

```diff
@@ -0,0 +1,14 @@
+name: Remove Labels
+
+on:
+  issues:
+    types: [closed]
+
+jobs:
+  remove_label:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions-ecosystem/action-remove-labels@v1
+        with:
+          labels: status:triage
```

Comment:
I'm hesitant to add a dependency on an external action — do you think you could use the `gh` CLI instead? I think that'd make it easier for us to extend this in the future too.